ansible/playbook/tasks/roles/nginx/files/conf.d/pmm.conf

```diff
@@ -48,6 +55,9 @@
     # Store the value of X-Must-Setup header of auth_request subrequest response in the variable.
     # It is used below in /auth_request.
     auth_request_set $auth_request_must_setup $upstream_http_x_must_setup;
+    # Store the value of X-Percona-Token header of auth_request subrequest response in the variable.
+    auth_request_set $auth_request_percona_token $upstream_http_x_percona_token;
+    proxy_set_header X-Percona-Token $auth_request_percona_token;  
```

Comment:
`X-Percona-Token` in my mind stands for the token to integrate with the percona portal. Maybe `X-VMGateway-Token`?

Another thing that bothers me but I can be wrong here. We set X-Percona-Token on every request proxied to any upstream we have. Can we move it to `location /prometheus/api/v1` maybe?

Reply:
@maxkondr any preference on the token name?

@gen1us2k good point re the url. I enabled it for all requests just to make sure it does not break any endpoint we did not cover in the spec. In the next version it will be enabled only for 4 endpoints we identified.
Is this something you would consider for this PR already?
The presence of the header shall not affect other functionality. My goal is to make sure `vmgateway` as a component does not break things.

Reply:
I have no strict objections regarding header name, but please:
- make it clear to understand that this is our custom header
- adjust the spec accordingly